README.md

```diff
@@ -67,7 +67,8 @@ PolicyFactory policy = new HtmlPolicyBuilder()
     .allowElements("p")
     .allowElements(
         new ElementPolicy() {
-          (String elementName, List<String> attrs) -> {
+          @Override
+          public String apply(String elementName, List<String> attrs) {
```

Comment:
Thanks for catching that.   Would it be sufficient to just get rid of the `new ElementPolicy` stuff and leave the lambda invocation?

Reply:
Sure.